### DISCUSSION TITLE
Why does source code include compilation?

### DISCUSSION THREAD

**Discussion by Gzushgshsh** [URUWorks/TeroSubtitler#126] [General]
If is not necessary for help contributors my suggestion is remove this and add a gitignore file

**Comment by URUWorks** [comment]
Can you explain me ?

**Comment by Gzushgshsh** [comment]
This build for mac: [https://github.com/URUWorks/TeroSubtitler/tree/main/TeroSubtitler/bin](url)
Why is it here in the GitHub repository?
I thought only source code should be here 🤔

**Comment by URUWorks** [comment]
Just to know the structure, I don't think it's a problem :)

**Comment by Gzushgshsh** [comment]
Right, I was just curious because I rarely see anyone doing that

Unrelated question:
I would like to make a localization for Brazilian Portuguese, where do I find the language files?

**Reply by URUWorks** [reply]
For example, make a copy of the file "en-US.xml", name it "pt-BR.xml" and translate the fields 😃
